### ISSUE TITLE
Graph Node start sync from block 0

### ISSUE THREAD

**Issue by vietanhduong** [graphprotocol/graph-node#3648]
**Do you want to request a *feature* or report a *bug*?**
* Feature

**What is the current behavior?**
Currently, `graph-node` start sync from block 0.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

**What is the expected behavior?**
We can tell the graph node what the genesis block is. Because, there are some chains, the genesis block start from 1.

```console
Jun 09 11:21:31.137 WARN Trying again after eth_getBlockByNumber(0, false) RPC call failed (attempt #10) with result Err(Ethereum node could not find genesis block), provider: velas-rpc-0
```

**Comment by khelaia** [comment]
add startBlock:
in your subgraph.yaml

**Comment by leoyvens** [comment]
Closed by #3650
